### DISCUSSION TITLE
[Virtualizer] Why do we do "pointerEvents: auto"?

### DISCUSSION THREAD

**Discussion by donovanlopez** [inokawa/virtua#503] [Q&A] [answered]
In the [Virtualizer](https://github.com/inokawa/virtua/blob/main/src/react/Virtualizer.tsx#L373), I see that when we're not scrolling, we do `pointerEvents: auto` and I was wondering why.

I ask because I'm using `Virtualizer` to create an infinite list where each item will have a `DropdownMenu` (from [Radix](https://www.radix-ui.com/primitives/docs/components/dropdown-menu)) but when you open a `DropdownMenu`, the expected behavior is that only the `DropdownMenu` should be in focus. However, because the `Virtualizer` wraps the items with the `Element` component with the `pointerEvents: auto` styling, the background elements become clickable even when the `DropdownMenu` is open (which is not desired).

**Comment by inokawa** [comment, marked as answer]
Setting `pointer-events: none` while scrolling is an optimization to disable hover interactions.
https://ryanseddon.com/css/pointer-events-60fps/

The `pointer-events: auto` is the opposite, but... is it better to just remove the style to inherit the parent's style? like [react-window](https://github.com/bvaughn/react-window/blob/8736ede0090a9652b81b7933affc52a26fe4d744/src/createListComponent.js#L385)? If so, I'll fix it later(or PR welcome).

**Reply by donovanlopez** [reply]
Hmmmm I feel like it'd make sense to inherit the parent's style. It'd be easier to suit the needs for different use cases.

**Reply by inokawa** [reply]
Released in `0.34.1`.
https://github.com/inokawa/virtua/pull/504

**Reply by donovanlopez** [reply]
Thanks!
